parts/linux/cloud-init/artifacts/components.json

```diff
@@ -131,7 +131,10 @@
       "multiArchVersions": [
         "6.8.7-main-04-09-2024-82adbf97",
         "6.8.7-main-04-09-2024-82adbf97-targetallocator",
-        "6.8.7-main-04-09-2024-82adbf97-cfg"
+        "6.8.7-main-04-09-2024-82adbf97-cfg",
```

Comment:
please make sure to come back and remove 6.8.7 after the 6.8.10 has finished

Reply:
Thanks Cameron! Yes sure, we will follow the same methodology as before. We will keep only 2 versions in the version list and will keep updating it.